sdk/keyvault/keyvault/src/keyVaultClient.ts

```diff
@@ -10,19 +10,19 @@
 
 import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
-import * as Models from "./models";
```

Comment:
I believe `keyVaultClient.ts` and `keyVaultClientContext.ts` are from autorest code-gen.  At one point I considered moving them into `src/generated` folder but that didn't happen.  Maybe we shouldn't change them.

Reply:
That was a mistake, I'll fix it.